TFT/src/User/Menu/UnifiedMove.c

```diff
@@ -18,18 +18,20 @@ void menuUnifiedMove(void)
       {ICON_EXTRUDE,                 LABEL_EXTRUDE},
       {ICON_DISABLE_STEPPERS,        LABEL_DISABLE_STEPPERS},
       {ICON_BABYSTEP,                LABEL_BABYSTEP},
-      #ifdef DELTA_PRINTER
-        {ICON_CALIBRATION,            ICON_CALIBRATION},
-      #else
-        {ICON_MANUAL_LEVEL,            LABEL_LEVELING},
-      #endif
+      {ICON_MANUAL_LEVEL,            LABEL_LEVELING},
       {ICON_BACKGROUND,              LABEL_BACKGROUND},
       {ICON_BACK,                    LABEL_BACK},
     }
   };
 
   KEY_VALUES key_num = KEY_IDLE;
 
+  if (DELTA_PRINTER)
+  {
+    UnifiedMoveItems.items[5].icon = ICON_CALIBRATION;
+    UnifiedMoveItems.items[5].label.index = LABEL_DELTA_CONFIGURATION;
```

Comment:
I have done these changes. Thanks!

Reply:
it has been fixed in #2067

Reply:
This PR fix the label, but not the logic to change the button, because "DELTA_PRINTER" is always defined and the expression "#ifdef DELTA_PRINTER" is always true.

Reply:
Yes #ifdef can't be used because DELTA_PRINTER is always defined and can be enabled with "true" parameters or disabled with "false".
@digant73 maybe you can change the logic in your PR to uncomment #define DELTA_PRINTER and #define REMOVABLE_PROBE when we want enable them and commented by default.